templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +26,12 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %} ">Update</a>
+              </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">Delete</a>
+              </td>
```

Comment:
Code Style Issue: There should be a 'Cancel' button in addition to the 'Delete' button. The 'Cancel' button should lead to the previous page the user was on.